webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -28,12 +28,11 @@ const Breadcrumb = ({
     <PfBreadcrumb {...props}>
       {items.map((item, index) => {
         const active = index === items.length - 1;
-        const {
-          caption,
-          caption: { icon, text },
-        } = item;
+        const { caption, url, onClick } = item;
+        const { icon, text } = caption || {};
+
         const overrideTitle = active && titleReplacement;
-        const itemTitle = overrideTitle || text || caption;
+        const itemTitle = overrideTitle || text || caption || '';
```

Comment:
If there is no title, does it make sense to add an item? Perhaps if there is no title and no icon, it should return nothing since the user can't click on anything anyway.

Reply:
How about returning `null` when there's no title nor icon?

Reply:
Updated, thanks!

Reply:
This needs to be reverted fo the `return` to work :thinking:

Reply:
That would still fail if `icon` is present but `itemTitle` is missing for the active element, since `EllipsisWithTooltip` can't handle `null` as its children

Reply:
Make sense :+1: